### DISCUSSION TITLE
Add support for ABC notation

### DISCUSSION THREAD

**Discussion by slimhk45** [quarto-dev/quarto-cli#2653] [Feature Requests]
There is already support for KaTeX in mathematics, it would be good as well to include support for ABC notation in music.

**Comment by mcanouil** [comment]
Seems like an easy Quarto extension to make for example using <https://paulrosen.github.io/abcjs/>

**Comment by cscheid** [comment]
It's not too hard, and I actually have a filter that I built for myself that uses abcjs. Let me see if I can make a nice abcjs extension.

**Reply by cscheid** [reply]
Update: I want to do this the right way, and it'll be better to wait until [this PR lands](https://github.com/quarto-dev/quarto-cli/pull/2521). Look for the extension in our upcoming 1.3 prereleases.

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/2700
